contracts/land/LANDdata.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.23;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+import "./LANDbase.sol";
+
+contract LANDdata is Ownable, LANDbase{
```

Comment:
Contract name is better to follow the coding style:

http://solidity.readthedocs.io/en/v0.4.24/style-guide.html#contract-and-library-names